feat: UTC-209: Limit project members in project/s endpoint.

robot-ci-heartex opened the pull request:
<!-- Follow Merge -->
This PR was automatically generated via Follow Merge. [Workflow Run](https://github.com/HumanSignal/label-studio/actions/runs/17083541143)
Please ensure that all linked upstream Pull Requests are merged before proceeding with this one.
- https://github.com/HumanSignal/label-studio-sdk/pull/512
<!-- Follow Merge -->

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17054566711

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17072163494

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-209'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17072163494

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17073231926

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17081208016

robot-ci-heartex committed:
Merge branch 'develop' into 'fb-utc-209'

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17081208016

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17081337691

mcanu commented:
/fm sync
> [Workflow run](https://github.com/HumanSignal/label-studio/actions/runs/17082061510)

mcanu committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17082061510

robot-ci-heartex committed:
Sync Follow Merge dependencies

Workflow run: https://github.com/HumanSignal/label-studio/actions/runs/17083541143
